impress-org/givewp
# Renewal payments are now included in reports.

Resolves #5398

## Description



Renewal payments weren't being included in the results of the `Give_Payments_Query` because they do not include a value for the `_give_payment_mode` meta.

Ideally, the renewal system would be updated to include this meta key/value, but that would require a database update to correct historical data.

Instead, I opted to add an additional query for renewal payments and combine the results.

## Affects



Updates the shared `Endpoint::getPayments()` to combine one-time and renewal payments.

## Pre-review Checklist




## Testing Instructions



1. Create a new donation form
1. Create a donation
1. Manually create a renewal
1. See Donations > Reports

Note: Due to #5399, you may need to manually adjust the start date to include first day's payments (for a fresh install).

The renewal payment is now listed in the reports.
